review/tests/test_review.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def review():
```

Comment:
The fixture, should create review or reviews objects
Why there is logic here? 

If there is a logic, you should test it, is'nt it?

Reply:
Fixed